lib/bank_account.rb

```diff
@@ -18,7 +18,7 @@ def initialize(id, balance, open_date, owner = nil)
     def withdraw(withdraw_amount)
       if @balance < withdraw_amount
         puts "Insufficient funds: withdraw denied. Your balance is #{@balance}."
-      elsif @balance >= withdraw_amount
+      elsif
       @balance = @balance.to_i - withdraw_amount.to_i
```

Comment:
Watch your indentation here - you'll want to have an additional indent within the `elsif` block